### DISCUSSION TITLE
how are you getting data from Codechef

### DISCUSSION THREAD

**Discussion by Sayemahamed** [naimulcsx/progHours#232] [General]
assalamualikum, Vhaiya amar jana moto Codechef kono API provide kora na . Codechef ar submission ki vhaba verification korta cen ?

**Comment by Imtiaz246** [comment]
Via web scraping. The submission pages are public in codechef.

**Reply by Sayemahamed** [reply]
library konta use korsen (scraping ar jonno )? lock file thake bujtaci na

**Reply by naimulcsx** [reply]
CodeChef has a legacy API, but they've stopped providing access to that. For parsing from the HTML, we're using cheerio.
